### PR TITLE
feat: form

### PR DESCRIPTION
**Pull request by benjycui**
close: #5056

close: #5055

### CODE REVIEW

**Comment by benjycui** [comment]
因为表单的响应式是使用 media query 做的，所以 [这部分逻辑](https://github.com/ant-design/ant-design/blob/master/components/form/FormItem.tsx#L224-L230) 没有做进去。

File: components/form/style/index.less [hunk 1]
```diff
@@ -247,47 +238,58 @@ form {
   }
 }
 
-// Form layout
-//== Vertical Form
-.@{form-prefix-cls}-vertical .@{form-prefix-cls}-item-label,
-.@{ant-prefix}-col-24.@{form-prefix-cls}-item-label {  // when labelCol is 24, it is a vertical form
```
Comment (afc163) [inline components/form/style/index.less]: Is this lost?

File: components/table/__tests__/__snapshots__/demo.test.js.snap [hunk 2]
```diff
@@ -813,7 +813,7 @@ exports[`test renders ./components/table/demo/dynamic-settings.md correctly 1`]
   <div
     class="components-table-demo-control-bar">
     <form
-      class="ant-form ant-form-inline">
+      class="ant-form ant-form-horizontal ant-form-inline">
```
Comment (afc163) [inline components/table/__tests__/__snapshots__/demo.test.js.snap]: Why both `horizontal` & `inline`

Comment (benjycui) [inline components/table/__tests__/__snapshots__/demo.test.js.snap]: fixed https://github.com/ant-design/ant-design/pull/5063/commits/d86c68494327a630feba9dc9e964980bbbdcf62b

Comment (benjycui) [inline components/table/__tests__/__snapshots__/demo.test.js.snap]: updated: 28fca28

File: components/form/style/index.less (hunk 1 above)
Comment (benjycui) [inline components/form/style/index.less]: fixed in 28fca28

**Comment by benjycui** [comment]
Updated

**Pull request merged**